src/api/app/assets/javascripts/webui2/kiwi_editor.js

```diff
@@ -215,6 +217,104 @@ function showOnAddition(list, show) { // jshint ignore:line
   });
 }
 
+function kiwiPackagesSetupAutocomplete(fields) {
```

Comment:
Would it be possible to build on top of what we already do with the other autocomplete fields?

Otherwise, we are duplicating code.